pkg/testutil/testutil.go

```diff
@@ -76,15 +77,19 @@ func Equals(tb testing.TB, exp, act interface{}, v ...interface{}) {
 	if len(v) > 0 {
 		msg = fmt.Sprintf(v[0].(string), v[1:]...)
 	}
-	tb.Fatal(sprintfWithLimit("\033[31m%s:%d:"+msg+"\n\n\texp: %#v\n\n\tgot: %#v%s\033[39m\n\n", filepath.Base(file), line, exp, act, diff(exp, act)))
+	tb.Fatalf("\033[31m%s:%d:"+msg+"\n\n\texp: %#v\n\n\tgot: %#v%s\033[39m\n\n", filepath.Base(file), line, trim(exp), trim(act), trim(diff(exp, act)))
 }
 
-func sprintfWithLimit(act string, v ...interface{}) string {
-	s := fmt.Sprintf(act, v...)
-	if len(s) > 1000 {
-		return s[:1000] + "...(output trimmed)"
+func trim(s interface{}) string {
+	str := fmt.Sprintf("%v", s)
+	_, noTrim := os.LookupEnv("NO_TRIM")
```

Comment:
What about just commenting out in code if you really need this. It's rare that trillions line of output is really useful - normally it just explodes your tests. :thinking:

Reply:
I would rather have it this way to switch quickly between trimmed and non-trimmed, otherwise if I change the code when needed will forget and add to a PR by mistake.

Do you see any problem with this approach?

Reply:
ping @bwplotka

Reply:
I would say the hidden envars like this tend to be 100% forgotten. I would say, let's increase size a little bit and work towards automatic printing only things that matters instead. If you put this accidently in a PR, we can just find it in code review (:

Reply:
ok